File: app/src/main/java/com/wiktor/weater1/newWeaher/NewWeatherActivity.java
package com.wiktor.weater1.newWeaher;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import com.wiktor.weater1.R;
import com.wiktor.weater1.newWeaher.cityList.mvp.CityListFragment;

import butterknife.BindString;
import butterknife.BindView;
import butterknife.ButterKnife;

public class NewWeatherActivity extends AppCompatActivity {

    @BindView(R.id.main_toolbar)
    Toolbar toolbar;
    @BindString(R.string.toolbar_title)
    String toolbarTitle;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_new_weather);

        ButterKnife.bind(this);
        setSupportActionBar(toolbar);
        openCityListFragment();

    }

    private void openCityListFragment() {
        CityListFragment cityListFragment = new CityListFragment();
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction transaction = fragmentManager.beginTransaction();
        transaction.replace(R.id.frame, cityListFragment);
        transaction.commit();
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item) {
        switch (item.getItemId()) {
            case android.R.id.home:
                onBackPressed();
                break;
        }
        return super.onOptionsItemSelected(item);
    }

    public void setMyTitle(String title) {
        toolbar.setTitle(title);
    }

    public void setMySubtitle(String subtitle) {
        toolbar.setSubtitle(subtitle);
    }

    public void showArrow(boolean show) {
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayHomeAsUpEnabled(show);
            getSupportActionBar().setDisplayShowHomeEnabled(show);
        }
    }
}

File: app/src/main/java/com/wiktor/weater1/newWeaher/NewWeatherActivity.kt
package com.wiktor.weater1.newWeaher

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.MenuItem
import com.wiktor.weater1.R
import com.wiktor.weater1.newWeaher.cityList.mvp.CityListFragment
import kotlinx.android.synthetic.main.toolbar.*

class NewWeatherActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_new_weather)

        setSupportActionBar(main_toolbar)
        openCityListFragment()
    }


    private fun openCityListFragment() {
        val cityListFragment = CityListFragment()
        val fragmentManager = supportFragmentManager
        val transaction = fragmentManager.beginTransaction()
        transaction.replace(R.id.frame, cityListFragment)
        transaction.commit()
    }


    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> onBackPressed()
        }
        return super.onOptionsItemSelected(item)
    }


    fun setMyTitle(title: String) {
        main_toolbar.title = title
    }

    fun setMySubtitle(subtitle: String) {
        main_toolbar.subtitle = subtitle
    }

    fun showArrow(show: Boolean) {
        supportActionBar?.let {
            it.setDisplayHomeAsUpEnabled(show)
            it.setDisplayShowHomeEnabled(show)
        }
    }
}
